Build/Test/cibuild.sh

```diff
@@ -28,21 +27,14 @@ if [ $? -eq "0" ]; then
     fi
 fi
 
-UNIT_BOOTSTRAP=".Build/vendor/typo3/cms/typo3/sysext/core/Build/UnitTestsBootstrap.php"
-if [[ $TYPO3_VERSION == "~8.7.0" ]]; then
-    UNIT_BOOTSTRAP=".Build/vendor/typo3/testing-framework/Resources/Core/Build/UnitTestsBootstrap.php"
-fi
-
 echo "Run unit tests"
-.Build/bin/phpunit --colors -c Build/Test/UnitTests.xml --coverage-clover=coverage.unit.clover --bootstrap=$UNIT_BOOTSTRAP
+UNIT_BOOTSTRAP=".Build/vendor/nimut/testing-framework/res/Configuration/UnitTestsBootstrap.php"
+.Build/bin/phpunit --colors -c Build/Test/UnitTests.xml --bootstrap=$UNIT_BOOTSTRAP
```

Comment:
In the line 12 the path is exported but not used at that place.
I guess because of`COMPOSER_BIN_DIR` the phpunit executable is in `$HOME/.composer/vendor/bin/` folder